### DISCUSSION TITLE
Welcome to singlestoredb-laravel-driver Discussions!

### DISCUSSION THREAD

**Discussion by carlsverre** [singlestore-labs/singlestoredb-laravel-driver#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to answer questions about the SingleStoreDB Laravel Driver that don't fit within Github Issues. We hope that you use this space to:
  * Ask questions you’re wondering about
  * Share ideas and projects that use the driver
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪

**Comment by DanielFerguson** [comment]
Is there any progress around geospatial development for the driver? 👀

**Reply by carlsverre** [reply]
No update at this point. Are there specific features or use cases you would like us to enable?
